### PR TITLE
2021-03-25 Update

### PR DESCRIPTION
**Pull request by EstebanMz**
- Change package name to "rva"
- Some brackets were reordered
- Updated event date to tonight's lobby

### CODE REVIEW

**Review by BGMP — COMMENTED** [review]
Now that I see the hardcoded event date, I really do believe this is not the place for the pack's .json reference to be stored at.

Any comments on that? Because if indeed we were to store it in here, a new commit must be pushed every day updating the event's date for the night's session.

**Comment by BGMP** [comment]
@Poputrash @Y100XTT Ideas?

**Comment by Poputrash** [comment]
I believe a daily schedule is unnecesary. Perhaps if it was monthly, it
would justify the amount of work required. I think this last one is the
safest bet, because is more work to move files to another repository
instead of simply modifying the schedule.
You can post a monthly schedule for events and such, and refer people to
the discord channel for details.

**Comment by EstebanMz** [comment]
If hosts were decided for a whole week and we only had to edit this file once per week it might be worthy to do this effort, as it happen at Re-Volt I/O, but it seems it is not the case in RVA. Honestly, I'd added the event feature to repo just because it was documented how to do it XD.
I vote to delete all the code related to events and only let the code about pack download through launcher.

File: rva-repo.json [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+    "events": 
+    {
+        "rva-lobby": 
+        {
+            "title": "Semi-Pro - Day 10",
+            "date": "26-03-2021 01:50",
+            "hosts": 
+            {
                 "Gforce": "187.75.55.183"
             },
```
Comment (BGMP) [inline rva-repo.json]:
```suggestion
```

Comment (BGMP) [reply rva-repo.json]: Remove the hosts section for now, I guess. We may automate/better organise for this feature eventually

**Review by BGMP — CHANGES_REQUESTED** [review]

**Review by BGMP — APPROVED** [review]

**Comment by BGMP** [comment]
Should be good to go!

**Pull request merged**